LibertyDSNP/example-client
# handle error UI when creating registrations and registrations now in …

Purpose
---------------
[https://www.pivotaltracker.com/story/show/179585398](https://www.pivotaltracker.com/story/show/179585398)
As a user, I want a more readable error handling UI when I am adding a new registration and searching for a new handle.


Solution
---------------
follow the Zeplin wireframes to display the error with more clear text and icons rather than just printing out the error message.
[https://zpl.io/aXyGmEp](https://zpl.io/aXyGmEp)

Change summary
---------------
* Style new error message when creating a registration
* add spinner
* save registrations in session storage

Steps to Verify
----------------
1. create and change handles
2. reload page to see handles stay in session storage
3. create handle for account with no registrations
4. create handle for account with existing registrations


Additional details / screenshot
----------------
** screen recording was cut off before the new registration loaded in at the end.
https://user-images.githubusercontent.com/43625033/134988578-27460711-e98b-4452-baf7-76e7329663cb.mov
